File: gforge/cronjobs/mail/add_alias.php
#!/usr/bin/php -f
<?php
$fp = fopen("php://stdin","r");
$fp2 = fopen("/etc/mail/aliases.mailman","a+");
if (!($fp)) {
	print ("ERROR: unable to open standard input\n");
	exit;
}
if (!($fp2)) {
	print ("ERROR: unable to open target file\n");
	exit;
}
while (!feof($fp)) {
	$alias = fgets($fp, 4096);
	fputs($fp2, $alias);
}
fclose($fp);
fclose($fp2);
?>

File: gforge/cronjobs/mail/add_alias.py
#!/usr/local/bin/python
import sys
import os
import string
import time
import commands

def main():
	fp2 = open("/etc/mail/aliases.mailman","a+")
	fp = sys.stdin
	while 1:
		alias = fp.readline()
		if not alias:
			break
#		print "writing alias", alias
		fp2.write(alias)
	fp.close()
	fp2.close()

if __name__ == '__main__':
    main()
